keyless/pepper/common/Cargo.toml

```diff
@@ -14,14 +14,25 @@ rust-version = { workspace = true }
 
 [dependencies]
 anyhow = { workspace = true }
+aptos-crypto = { workspace = true }
+aptos-crypto-derive = { workspace = true }
+aptos-dkg = { workspace = true }
```

Comment:
Do you need `aptos-dkg`?

Reply:
yes, I use the bls pairing function from there I believe